README.md

```diff
@@ -4,136 +4,163 @@
 
 **Notice: This project is still in beta, under active development. Features and compatibility may change.**
 
-## Building
+* [Usage](#usage)
+   * [Tracing nginx](#nginx)
+      * [Quick start/Example](#example)
+      * [Guide](#guide)
+* [Development](#building)
 
-**Dependencies**
+<a name="usage"/>
```

Comment:
You don't need to manually add anchors.
Github will munge them to something different anyway.